### PR TITLE
feat: trigger table Icons

### PR DESCRIPTION
**Pull request by RonenMars**
## Description

- If connection: display the connection name with icon'
- if scheduler / webhook display Scheduler / Webhook with icon
- If webhook: Display also a copy icon to copy the webhook - hover on copy will display the webhook

image

## Linear Ticket
https://linear.app/autokitteh/issue/UI-834/triggers-table-source-content-change

## What type of PR is this? (check all applicable)

-   [x] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/triggers/table.tsx [hunk 1]
```diff
@@ -30,129 +96,108 @@ export const TriggersTable = () => {
...
-									className="opacity-0 group-hover:opacity-100"
-									isActive={"entrypoint" === sortConfig.key}
-									sortDirection={sortConfig.direction}
-								/>
-							</Th>
-
-							<Th className="w-1/4 max-w-20 text-right font-normal">{t("table.columns.actions")}</Th>
+							{tableHeaders.map(({ className, key, label, sortable }) => (
+								<Th
+									className={`group ${sortable ? "cursor-pointer" : ""} ${className} font-normal`}
```
Comment (J1za) [inline src/components/organisms/triggers/table.tsx:171]: use `cn` lib

File: src/components/organisms/triggers/table.tsx [hunk 2]
```diff
@@ -30,129 +96,108 @@ export const TriggersTable = () => {
...
-		addToast({
-			message: t("table.triggerRemovedSuccessfully"),
-			type: "success",
-		});
-		LoggerService.info(namespaces.ui.triggers, t("table.triggerRemovedSuccessfullyExtended", { triggerId }));
-
-		fetchTriggers(projectId!, true);
-		setTriggerId(undefined);
-	};
+	}, [triggerId, projectId, t, tError, addToast, closeModal, fetchTriggers]);
```
Comment (J1za) [inline src/components/organisms/triggers/table.tsx:137]:
```suggestion
	}, [triggerId, projectId]);
```

File: src/components/organisms/triggers/table.tsx (hunk 1 above)
Comment (RonenMars) [reply src/components/organisms/triggers/table.tsx:171]: refactored

File: src/components/organisms/triggers/table.tsx (hunk 2 above)
Comment (RonenMars) [reply src/components/organisms/triggers/table.tsx:137]: Done

**Pull request merged**